From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@TwoUnder__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mausher). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

JennieHubbard commented:
#sign-off

rmca14 commented:
@barbkess

WilliamAntonRohm committed:
initial safety commit

WilliamAntonRohm committed:
renaming

WilliamAntonRohm committed:
initial editing

WilliamAntonRohm committed:
should-be final edits

WilliamAntonRohm committed:
Merge branch 'master' of https://github.com/WilliamAntonRohm/azure-docs-pr into W-hdi-install-pub-apps

lajkdf
ljkad
the commit.

davidmu1 committed:
new vmss tutorial

WilliamAntonRohm committed:
final edits

WilliamAntonRohm committed:
resized & added borders as warranted

WilliamAntonRohm committed:
adding all TOCs

davidmu1 committed:
revised per feedback

davidmu1 committed:
new create tutorials

davidmu1 committed:
acrolinx corrections

davidmu1 committed:
acrolinx corrections

davidmu1 committed:
acrolinx corrections

davidmu1 committed:
new waf tutorial

davidmu1 committed:
acrolinx correction

davidmu1 committed:
acrolinx correction

ellacroi committed:
Initial Draft for Marketplace Publishing Guide

ellacroi committed:
added sections commit

ellacroi committed:
xxx

ellacroi committed:
xxx

WilliamAntonRohm committed:
updated author and date

ellacroi committed:
almost complete

ellacroi committed:
almost done

ellacroi committed:
still trying to get bash to work

paulathurman committed:
edit pass: data-factory/connector-batch1

ellacroi committed:
fixing tables and images

ellacroi committed:
fixing tables and images

ellacroi committed:
image fix

ellacroi committed:
image fix again git bash crashed

ellacroi committed:
Third try with the image

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-factory/connector-batch1

paulathurman committed:
edit pass: data-factory/connector-batch1

msmbaldwin committed:
Adding new marketplace publisher's guide

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

ellacroi committed:
Merge pull request #1 from msmbaldwin/marketplace-articles

Adding new marketplace publisher's guide

ellacroi committed:
Removed unneeded images

ellacroi committed:
Remove unneeded images

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

kanshiG committed:
Update performance-tips.md

Add client side metrics for retrys of the request - which indicates reason for extra latency

paulathurman committed:
fixing merge conflicts

ellacroi committed:
hopefully final in first draft

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

paulathurman committed:
edit pass: data-factory/connector-batch1

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

ellacroi committed:
trying to get rid of links

ellacroi committed:
changing based on PR review feedback

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

ellacroi committed:
Wrapping up feedback from PR

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

Commit:
acrolinx updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into imageupdates

itechedit committed:
edit pass: active-directory-saas-access-panel-user-help

itechedit committed:
Update active-directory-saas-access-panel-user-help.md

kanshiG committed:
Update performance-tips.md

Removed extra space and updated doc line

cforbe committed:
Calling out Azure Comos DB as a data source

Management asked for this to be separated out from it's original appendix to be a new stand alone appendix

cforbe committed:
Update toc.yml

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

mimig1 committed:
Indenting new content to get continuous numbering

ellacroi committed:
updated TOC

ellacroi committed:
updated TOC

ellacroi committed:
updated TOC

ellacroi committed:
Fixed table

MikeDodaro committed:
Added 3 files for SDK topics + modify TOCs

Commit:
automation : image updates

bwren committed:
activity

ellacroi committed:
incorporating second round of PR feedback

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

ellacroi committed:
Fixing images and spacing after callouts

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into marketplace-publishing

itechedit committed:
Update active-directory-saas-access-panel-user-help.md

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-factory/connector-batch1

cherylmc committed:
update

paulathurman committed:
edit pass: data-factory/connector-batch1

cherylmc committed:
resize

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-factory/connector-batch1

paulathurman committed:
edit pass: data-factory/connector-batch1

bwren committed:
ad replication

bwren committed:
capacity

v-geberr committed:
region as part of key name

mimig1 committed:
Cosmos DB: Adding Kafka HDInsight article to TOC

adegeo committed:
fixed title

Commit:
updates

kanshiG committed:
Merge pull request #2 from mimig1/patch-199

Indenting new content to get continuous numbering

Commit:
Merge branch 'W-hdi-install-pub-apps' of https://github.com/WilliamAntonRohm/azure-docs-pr into hdi-new-docs

rwike77 committed:
updating for certs, removing unused graphic

Commit:
adding docs to toc

Commit:
correcting toc

Commit:
updates

rwike77 committed:
updated for cert

rwike77 committed:
updated for cert

Commit:
image updates

ShawnJackson committed:
Merge pull request #31993 from itechedit/active-directory-saas-access-panel-user-help

edit pass: active-directory-saas-access-panel-user-help

rwike77 committed:
more info

v-shils committed:
Merge pull request #31630 from ellacroi/marketplace-publishing

Marketplace publishing

ShawnJackson committed:
Update connector-azure-blob-storage.md

MikeDodaro committed:
Editorial review changes

ShawnJackson committed:
Update connector-azure-table-storage.md

ShawnJackson committed:
Update connector-dynamics-crm-office-365.md

v-kents committed:
toc converted to yml

ShawnJackson committed:
Merge pull request #31723 from paulathurman/data-factory/connector-batch1

edit pass: Data factory/connector batch1

v-shils committed:
Merge pull request #32155 from cjgronlund/hdi-new-docs

new "Install app XYZ on HDInsight" articles from Dev Guide

v-geberr committed:
clarified Bing spell check requirements

adegeo committed:
fixed header

cforbe committed:
Update toc.yml

cforbe committed:
Update and rename data-prep-load-azure-cosmos-db-into-data-prep.md to data-prep-load-azure-cosmos-db.md

cforbe committed:
Update data-prep-appendix8-sample-source-connections-python.md

curtand committed:
update with permissions and fix screenshot

cforbe committed:
Update data-prep-appendix8-sample-source-connections-python.md

daveba committed:
adding link to event hubs on msi overview page

Lisaco88 committed:
Update sdk.md

Lisaco88 committed:
Update sdk.md

Lisaco88 committed:
Update sdk.md

Lisaco88 committed:
Update sdk.md

davidmu1 committed:
new quickstarts

PRMerger19 committed:
Merge pull request #32168 from MicrosoftDocs/add-event-hub-link-msi-overview-page

adding link to event hubs on msi overview page

MikeDodaro committed:
Fix path conflicts

curtand committed:
typo

PRMerger12 committed:
Merge pull request #32153 from Thraka/apicert

fixed title

curtand committed:
removed User Account Administrator

davidmu1 committed:
revised TOC

Commit:
Merge pull request #31678 from kanshiG/patch-25

Update performance-tips.md

MikeDodaro committed:
Resolved conflicts

curtand committed:
removed User Account Administrator

MikeDodaro committed:
Remove : in line23 - three files

v-shils committed:
Merge pull request #32081 from georgewallace/imageupdates

automation : image update #2

adegeo committed:
fixed toc error grrr

curtand committed:
update per Kairaz fedback

v-shils committed:
Merge pull request #32159 from rwike77/quickstart

Quickstart updates

mmacy committed:
[acr] service principal script correction

rmca14 committed:
Merge pull request #32151 from v-geberr/0125-add-key-region

region as part of key name

davidmu1 committed:
minor cleanup

mmacy committed:
[acr] minor comment change

ktoliver committed:
Merge pull request #32160 from v-kents/1160220

replaced service-fabric/toc.md with toc.yml

MikeDodaro committed:
Fix colon in video, image, news

curtand committed:
removed one User permission

Commit:
Merge pull request #32173 from mmacy/acr-script-fix

[acr] service principal script correction

rmca14 committed:
Update data-prep-load-azure-cosmos-db.md

ktoliver committed:
Merge pull request #32162 from v-geberr/0125-bingspellcheck

clarified Bing spell check requirements

Commit:
Merge pull request #32177 from MikeDodaro/fixColon3files

Fix colon in video, image, news

v-shils committed:
Merge pull request #32170 from bwren/la-oms-solutions

Remove OMS from Log Analytics - solutions

MikeDodaro committed:
Change to correct conflict

rmca14 committed:
Merge pull request #32152 from mimig1/01252018toc

Cosmos DB: Adding Kafka HDInsight article to TOC

MikeDodaro committed:
Merge branch 'master' into addSDKpages

wmgries committed:
Convert ZRS blurb into includes.

rmca14 committed:
Merge pull request #32041 from cforbe/master

Calling out Azure Comos DB as a data source

ktoliver committed:
Merge pull request #32172 from Thraka/tocfix

fixed toc error grrr

wmgries committed:
Azure Files supports ZRS.

rwike77 committed:
updating quickstart

davidmu1 committed:
minor corrections

wmgries committed:
Fix typo in includes link.

v-shils committed:
Merge pull request #32169 from davidmu1/quickstartsappgateway

new quickstarts

rwike77 committed:
updates

davidmu1 committed:
minor corrections

wmgries committed:
Fix meaningless Acrolinx issue to force revalidation.

Lisaco88 committed:
Merge pull request #32060 from MikeDodaro/addSDKpages

Added 3 files for SDK topics + modify TOCs

Commit:
task 853: For create endpoint, update steps and screenshots

davidmu1 committed:
Merge branch 'master' into tutorialcreate

Ja-Dunn committed:
Merge pull request #32113 from cherylmc/active4

update

Commit:
task 853: Fix step spacing

Ja-Dunn committed:
Merge pull request #32166 from curtand/kairaz0125

update with permissions and fix screenshot

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #30341 from davidmu1/tutorialwaf

new waf tutorial

Ja-Dunn committed:
Merge pull request #32179 from wmgries/files-zrs

Update for Files to support ZRS

Ja-Dunn committed:
Merge pull request #32180 from rwike77/freshness

updating quickstart

v-alje committed:
Merge pull request #32186 from MicrosoftDocs/FromPublicRepo

From public repo

Lisaco88 committed:
Merge pull request #30305 from davidmu1/tutorialcreate

Tutorialcreate

Ja-Dunn committed:
Merge pull request #32183 from dksimpson/dean-853

task 853: For create endpoint, update steps and screenshots
